src/main/kotlin/objectmappings/mapper/ContactMapperKt.kt

```diff
@@ -3,19 +3,17 @@ package objectmappings.mapper
 import io.saisuryak.lab.objectmappings.model.datatransferobject.ContactInfoDTO
 import io.saisuryak.lab.objectmappings.model.domainobject.ContactInfoDO
 
-import java.util.stream.Collectors.toList
-
 object ContactMapperKt {
 
     fun contactDOToContactDTO(contactInfoDO: ContactInfoDO): ContactInfoDTO {
-        val contactInfoDTO = ContactInfoDTO()
-        contactInfoDTO.type = contactInfoDO.type.toString()
-        contactInfoDTO.value = contactInfoDO.value
-        contactInfoDTO.isActive = contactInfoDO.isActive
-        return contactInfoDTO
+        return ContactInfoDTO().apply {
```

Comment:
I'm excited to see the byte code generated for this, coz the POJOs are in java, I want to see how named arguments are converted to constructor style